common-theurgy.lic

```diff
@@ -142,6 +142,20 @@ module DRCTH
     DRCI.put_away_item?(DRC.left_hand, container)
   end
 
+  def sprinkle_holy_water?(theurgy_supply_container, water_holder, target)
+    empty_cleric_hands(theurgy_supply_container)
```

Comment:
Why empty hand first?

Reply:
I thought we needed to, but we don't. It works just the same on the left hand. Tested just now.